src/components/Description/description.jsx

```diff
@@ -0,0 +1,27 @@
+
+export default function Description(){
+    return (
+    <div class='Description'>
+        <h1 class='header'>
+            style={{
+                color: "#000000",
+                fontWeight: 'bold',
+                fontSize: 38
+            }}
+            Description
+            {"\n"}
+        </h1>
+        <Text numberOfLines={3}>
+            style={{
```

Comment:
I am not certain why we are setting color, I don't believe colors other than black and white were used for this